libecole/src/scip/model.cpp

```diff
@@ -185,6 +216,30 @@ std::map<std::string, Param> Model::get_params() const {
 	return name_values;
 }
 
+std::map<std::string, Param> Model::pause_limits() {
+	static auto constexpr names = std::array{
+		"limits/time",
+		"limits/nodes",
+		"limits/totalnodes",
+		"limits/stallnodes",
+		"limits/memory",
+		"limits/gap",
+		"limits/absgap",
+		"limits/solutions",
+		"limits/bestsol",
+		"limits/maxsol",
+		"limits/maxorigsol",
+		"limits/restarts",
+		"limits/autorestartnodes",
+	};
+	auto saved = std::map<std::string, Param>{};
+	for (auto const& name : names) {
+		saved[name] = get_param<Param>(name);
+		set_param(name, get_param_default<Param>(name));
```

Comment:
Also, this code assumes that the default time limits are infinite. Is it always the case ?

Reply:
I checked and yes, except for two limits that, I hadn't realized, don't need to be included in the list, since they don't stop the solving.

```
# maximal number of solutions to store in the solution storage
# [type: int, advanced: FALSE, range: [1,2147483647], default: 100]
limits/maxsol = 100

# maximal number of solutions candidates to store in the solution storage of the original problem
# [type: int, advanced: FALSE, range: [0,2147483647], default: 10]
limits/maxorigsol = 10
```

Reply:
Moved to separate PR.